public_html/wp-content/plugins/wordcamp-coming-soon-page/classes/wordcamp-coming-soon-page.php

```diff
@@ -466,6 +475,31 @@ public function block_new_post_admin_notice() {
 		}
 	}
 
+	/**
+	 * Show a notice if Coming Soon is enabled also in Gutenberg block editor.
+	 */
+	public function maybe_show_block_editor_notice() {
```

Comment:
The gutenberg notice is also showing up on pages, organizers, etc. It should probably only show up on posts.